x/staking/keeper/unbonding.go

```diff
@@ -59,18 +57,14 @@ func (k Keeper) GetUnbondingDelegationByUnbondingID(ctx context.Context, id uint
 		return types.UnbondingDelegation{}, types.ErrNoUnbondingDelegation
 	}
 
-	value, err := store.Get(ubdKey)
-	if err != nil {
-		return types.UnbondingDelegation{}, err
-	}
+	delAddr := ubdKey[2 : (len(ubdKey)/2)+1]
```

Comment:
Could we get a small godoc explaining the split. Helps with future developers when they run across this. Not sure if there is a go doc elsewhere